test/db/tools/rz_find

```diff
@@ -262,10 +262,8 @@ RUN
 
 NAME=rz-find -E with echo as a command
 FILE==
-CMDS=!rz-find -s "README" -E "echo" bins/arm
+CMDS=!rz-find -s "README" -E "echo" bins/arm~README~?
 EXPECT=<<EOF
-
-5
-0x0
+0
```

Comment:
this should return 1, not 0

Reply:
I fixed this, it was because rizin doesn't have support for nested Greps so I fixed it to one internal grep but now issue is something else, it works as expected on Linux but it's returning `2` on windows.